### PR TITLE
Add photo module to test db connection

### PR DESCRIPTION
**Pull request by hunter-s70**

### CODE REVIEW

File: src/photos/photo.controller.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import { PhotoService } from './photo.service';
+
+@Controller('photos')
+export class PhotoController {
+  constructor(private readonly photoService: PhotoService) {}
+
+  @Get()
+  async getPhotos(): Promise<any> {
+    // const user = await this.photoService.createPhoto();
+    return this.photoService.findAll().then((data) => {
```
Comment (haltaction) [inline src/photos/photo.controller.ts:11]: Try to use async/await approach

File: src/photos/photo.controller.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import { Controller, Get } from '@nestjs/common';
+import { PhotoService } from './photo.service';
+
+@Controller('photos')
+export class PhotoController {
+  constructor(private readonly photoService: PhotoService) {}
+
+  @Get()
+  async getPhotos(): Promise<any> {
```
Comment (haltaction) [inline src/photos/photo.controller.ts:9]: Probably return type of Promise will be something like `Promise<Array<Photo>>`
In the future better to try not to use **any**, only in extraordinary cases.

File: src/photos/photo.controller.ts (hunk 1 above)
Comment (hunter-s70) [reply src/photos/photo.controller.ts:11]: @haltaction I'll focus on it in future

**Comment by hunter-s70** [comment]
This code is not related to the auth feature. It was used for testing

**Pull request closed without merging**